build.rs

```diff
@@ -0,0 +1,116 @@
+#[macro_use]
+extern crate lazy_static;
+
+use std::fmt;
+use std::path::Path;
+use std::fs::{File, read_dir, metadata};
+use std::io::{Write, BufWriter};
+
+// Location of the dist folder
+#[cfg(not(target_os = "windows"))]
+const DIST: &str = "frontend/dist/";
+#[cfg(target_os = "windows")]
+const DIST: &str = "frontend\\dist\\";
```

Comment:
I wasn't sure given the nature of that's how it normally does paths. I didn't even know raw strings were a thing. I knew about byte ones